CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to the easyphysics package
+
+We appreciate your time and effort in contributing to this project. Any contributions are welcomed. These contributions include but are not limited to:
+
+**Report or fix bugs**
+
+**Fix wrong analysis and typos**
+
+**Report bugs or fix bugs**
+
+**Provide feedback**
+
+**Other meaningful input**
+
+## How to contribute to this project
+
+### GitHub Issue
+
+If you want to have any type of contributions to this package, please create a GitHub issue in our repository to inform us. Then please follow the instructions below for different types of contributions.
+
+### Code Contribution
+
+Please set up `easyphysics` for local development.
```

Comment:
@YHuUBC should this be easyphysicsR (and not easyphysics)?

Reply:
Right. Thank you for catching that!